.circleci/config.yml

```diff
@@ -271,26 +271,28 @@ workflows:
             branches:
               only: master
   check-and-deploy-staging-and-production:
-      jobs:
+    jobs:
       - check-code-formatting:
           context: api-nuget-token-context
-      - build-and-test:
-          context:
-              - api-nuget-token-context
-              - SonarCloud
+      - assume-role-staging:
+          context: api-assume-role-housing-staging-context
+          requires:
```

Comment:
The assume-role-staging and check-code-formatting jobs can run at the same time. Please remove lines 279 and 280.

Reply:
Removed.

There are two reasons for failing tests 
- no SSM permissions for build-and-test step on staging, this patch will fix it
- expired certificates

Anyway will wait till certificates are renewed.